llama2.mojo

```diff
@@ -977,8 +974,10 @@ fn main() raises:
                 next_token = argmax(state.logits)
             else:
                 # Apply the temperature to the logits
-                for q in range(config.vocab_size):
+                @parameter
+                fn temp(q:Int):
```

Comment:
It make no sense to parallelize single operation (division "/")
much better to vectorize, would you mind implement it ? 
Also do some benchmarks. It doesn't make sense to implement optimization if there is no difference in execution time or "tok/s" performance